src/anemoi/training/train/forecaster.py

```diff
@@ -201,18 +218,26 @@ def _step(
     ) -> tuple[torch.Tensor, Mapping[str, torch.Tensor]]:
         del batch_idx
         loss = torch.zeros(1, dtype=batch.dtype, device=self.device, requires_grad=False)
-        batch = self.model.pre_processors(batch)  # normalized in-place
+        # for validation not normalized in-place because remappers cannot be applied in-place
+        batch = (
+            self.model.pre_processors(batch, in_place=False) if validation_mode else self.model.pre_processors(batch)
```

Comment:
could just be something like `in_place=not validation_mode` instead

Reply:
You are right, this is a lot cleaner.